src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -115,14 +115,19 @@ export function TravellerSelection({
         selectionMode={selectionMode}
         fareProductType={fareProductType}
         selectableUserProfilesWithCountInit={userProfilesState}
-        setOnBehalfOfToggle={setOnBehalfOfToggle}
         isOnBehalfOfToggle={isOnBehalfOfToggle}
         close={(
           chosenSelectableUserProfilesWithCounts?: UserProfileWithCount[],
+          onBehalfOfToggle?: boolean,
         ) => {
           if (chosenSelectableUserProfilesWithCounts !== undefined) {
             setUserProfilesState(chosenSelectableUserProfilesWithCounts);
           }
+          if (onBehalfOfToggle !== undefined) {
+            setOnBehalfOfToggle(onBehalfOfToggle);
+          } else {
+            setOnBehalfOfToggle(isOnBehalfOfToggle);
+          }
```

Comment:
```suggestion
          }
```
This shouldn't be necessary as it will only set `isOnBehalfOfToggle`=`isOnBehalfOfToggle`.

Maybe `setOnBehalfOfToggle` should be called `setIsOnBehalfOfToggle` to make it even more clear

Reply:
updated!